Source/AnchorpointCli/Public/AnchorpointCliStatus.h

```diff
@@ -79,4 +79,43 @@ struct ANCHORPOINTCLI_API FAnchorpointHistoryEntry
 	FDateTime Date;
 };
 
-using FAnchorpointHistory = TArray<FAnchorpointHistoryEntry>;
\ No newline at end of file
+using FAnchorpointHistory = TArray<FAnchorpointHistoryEntry>;
+
+/**
+ * Copy of FGitConflictStatusParser for Anchorpoint
+ * 
+ * Extract the status of a unmerged (conflict) file
+ *
+ * Example output of git ls-files --unmerged Content/Blueprints/BP_Test.uasset
+100644 d9b33098273547b57c0af314136f35b494e16dcb 1	Content/Blueprints/BP_Test.uasset
+100644 a14347dc3b589b78fb19ba62a7e3982f343718bc 2	Content/Blueprints/BP_Test.uasset
+100644 f3137a7167c840847cd7bd2bf07eefbfb2d9bcd2 3	Content/Blueprints/BP_Test.uasset
+ *
+ * 1: The "common ancestor" of the file (the version of the file that both the current and other branch originated from).
+ * 2: The version from the current branch (the master branch in this case).
+ * 3: The version from the other branch (the test branch)
+ */
+class ANCHORPOINTCLI_API FAnchorpointConflictStatus
+{
+public:
+	/** Parse the unmerge status: extract the base SHA1 identifier of the file */
+	FAnchorpointConflictStatus(const TArray<FString>& InResults)
+	{
+		const FString& CommonAncestor = InResults[0]; // 1: The common ancestor of merged branches
+		CommonAncestorFileId = CommonAncestor.Mid(7, 40);
```

Comment:
it might make sense to split the line by spaces because Git now supports SHA256 mode in which case the IDs are much longer